pkgs/development/compilers/bs-platform/build-bs-platform.nix

```diff
@@ -41,10 +47,13 @@ stdenv.mkDerivation {
 
   installPhase = ''
     mkdir -p $out/bin
-    cp -rf jscomp lib vendor odoc_gen native $out
+    cp -rf jscomp lib ${bin_folder} vendor odoc_gen native bsb bsc $out
+    mkdir $out/lib/ocaml
+    cp jscomp/runtime/js.* jscomp/runtime/*.cm* $out/lib/ocaml
+    cp jscomp/others/*.ml jscomp/others/*.mli jscomp/others/*.cm* $out/lib/ocaml
+    cp jscomp/stdlib-406/*.ml jscomp/stdlib-406/*.mli jscomp/stdlib-406/*.cm* $out/lib/ocaml
     cp bsconfig.json package.json $out
-    ln -s $out/lib/bsb $out/bin/bsb
-    ln -s $out/lib/bsc $out/bin/bsc
-    ln -s $out/lib/bsrefmt $out/bin/bsrefmt
```

Comment:
bsrefmt is gone?

Reply:
This is probably an oversight. I’ll check soon!